reflex-dom-core/test/gc.hs

```diff
@@ -38,6 +38,7 @@ main = do
         postBuild <- getPostBuild
         let f n = liftIO $ if n < 3000
               then do performMajorGC
+                      threadDelay 1000 -- Wait a bit to allow requestAnimationFrame to call its callback sometimes; this value was experimentally determined
```

Comment:
But wouldn't this value be different for different computers, and different CPU load?